.github/workflows/deploy.yml

```diff
@@ -34,11 +37,36 @@ jobs:
             echo "HEROKU_APP_NAME=${{ secrets.HEROKU_APP_NAME }}-staging" >> $GITHUB_ENV
           fi
 
-      - uses: akhileshns/heroku-deploy@v3.7.8
-        with:
-          heroku_api_key: ${{ secrets.HEROKU_API_KEY }}
-          heroku_app_name: $HEROKU_APP_NAME
-          heroku_email: ${{ secrets.HEROKU_EMAIL }}
-          usedocker: true
+      - name: Set staging database url
+        if: ${{ env.BRANCH_TAG != 'latest' }}
         env:
-          BEEGO_ENV: 'prod'
+          DATABASE_URL: ${{ secrets.DATABASE_URL_STAGING }}
+        run: |
+          echo "DATABASE_URL=$DATABASE_URL" >> $GITHUB_ENV
+
+      - name: Set production database url
+        if: ${{ env.BRANCH_TAG == 'latest' }}
+        env:
+          DATABASE_URL: ${{ secrets.DATABASE_URL_PRODUCTION }}
+        run: |
+          echo "DATABASE_URL=$DATABASE_URL" >> $GITHUB_ENV
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Build Docker image
+        run: |
+          docker-compose pull || true
+          docker-compose build --build-arg DATABASE_URL=$DATABASE_URL
+          docker-compose push app
+
+      - name: Log in to Heroku container
+        run: heroku container:login
+
+      - name: Publish application
+        run: |
+          heroku container:push --recursive -a $HEROKU_APP_NAME
+          heroku container:release web worker -a $HEROKU_APP_NAME
```

Comment:
Does `heroku container:push` and `heroku container:release` looking for `Dockerfile` instead of `Dockerfile.web` for web process? I found [this](https://devcenter.heroku.com/articles/container-registry-and-runtime#pushing-multiple-images) in the heroku documentation but still not sure.

Reply:
For `web` process it looks for `Dockerfile` and for `worker` it looks for `Dockerfile.web`